python/dgl/sparse/sddmm.py

```diff
@@ -58,32 +58,33 @@ def sddmm(A: SparseMatrix, X1: torch.Tensor, X2: torch.Tensor) -> SparseMatrix:
 def bsddmm(A: SparseMatrix, X1: torch.Tensor, X2: torch.Tensor) -> SparseMatrix:
     r"""Sampled-Dense-Dense Matrix Multiplication (SDDMM) by batches.
 
-    ``sddmm`` multiplies two dense matrices :attr:`X1` and :attr:`X2` at the
-    nonzero locations of sparse matrix :attr:`A`. Values of :attr:`A` is not
-    considered during the computation.
+    ``sddmm`` matrix-multiplies two dense matrices :attr:`X1` and :attr:`X2`,
+    then elementwise-multiplies the result with sparse matrix :attr:`A` at the
+    nonzero locations.
 
     Mathematically ``sddmm`` is formulated as:
 
     .. math::
         out = (X1 @ X2) * A
 
-    The batch dimension is the last dimension for input matrices. In particular,
-    if the sparse matrix has scalar non-zero values, it will be broadcasted
-    for bsddmm.
+    The batch dimension is the last dimension for input dense matrices. In
+    particular, if the sparse matrix has scalar non-zero values, it will be
+    broadcasted for bsddmm.
 
     Parameters
     ----------
     A : SparseMatrix
-        Sparse matrix of shape ``(M, N)`` or ``(M, N, B)``.
+        Sparse matrix of shape ``(L, N)`` with scalar values or vector values of
```

Comment:
```suggestion
        Sparse matrix of shape ``(L, N)`` with vector values of length ``K``.
```

Reply:
I think we support broadcast, which allows scalar values in sparse matrix as well?